cmd/mdmctl/apply.go

```diff
@@ -68,9 +78,90 @@ Examples:
 
 func (cmd *applyCommand) applyBlueprint(args []string) error {
 	flagset := flag.NewFlagSet("blueprints", flag.ExitOnError)
+	var (
+		flBlueprintPath    = flagset.String("f", "", "filename of blueprint JSON to apply")
+		flNewBlueprintPath = flagset.String("n", "", "filename of new blueprint JSON to create")
```

Comment:
I agree with you that it can be helpful, but I think `-n` vs `-f` is confusing. Can we rename the `-n` flag and help text to match the fact that we're generating a template for the user to edit? Maybe `-generate-blueprint`?